addons/project_sms/tests/test_project_sharing.py

```diff
@@ -75,15 +75,15 @@ def test_project_user_can_change_stage_with_sms_template(self):
 
             The sms template should be sent and the stage should be changed on the task.
         """
-        project_user_group = self.env.ref('project.group_project_user')
-        sale_manager_group = self.env.ref('sales_team.group_sale_manager', False)
+        project_user_group, sale_manager_group = self.env.ref(
+            'project.group_project_user',
+            'sales_team.group_sale_manager',
+            raise_if_not_found=False
```

Comment:
Only one is optional, so you need 2 calls here.

Reply:
I think it doesn't matter much, since `project.group_project_user` is already checked in the `setupClass` and set on `self.user_projectuser`
Also, if `project.group_project_user` is not installed on `project_sms`, we have worse consistency problems than this test.

![immagine](https://github.com/user-attachments/assets/5bd337b0-6652-4455-9f13-5a3746f24d6b)